src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const DAY_COST = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUN = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUN = 20;
+  const priceWithoutDiscount = days * DAY_COST;
+  let price = priceWithoutDiscount;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    price = price - SHORT_TERM_DISCOUN;
+  }
+
+  if (days >= LONG_TERM) {
+    price = price - LONG_TERM_DISCOUN;
+  }
```

Comment:
This works perfectly fine but you can simplify it.
Instead of checking for short term first, check the long term
```suggestion
  if (days >= LONG_TERM) {
  // ...
  }

  if (days >= SHORT_TERM) {
    // ...
  }
```

If the days value is greater than LONG_TERM, it will get inside of this if statement.
If it is not greater than LONG_TERM but greater than SHORT_TERM, it will get inside of second if statement.
If the value is less than both LONG & SHORT TERM, you return `basePrice` at the bottom of the function